src/compositor/input/pointer.rs

```diff
@@ -147,7 +171,6 @@ fn focus_under_pointer<'view, V>(seat: &mut compositor::Seat,
                                       (keyboard: {keyboard})] => {
                     match shell.state() {
                         Some(&mut TopLevel(ref mut toplevel)) => {
-                            // TODO Don't send this for each keyboard!
```

Comment:
This has no performance impact.

Reply:
This is not a performance thing. We don't want to set all the views activated, only the focused one. The others should be deactivated